proto/install.sh

```diff
@@ -25,7 +25,7 @@ fi
 
 file=protoc-${version}-${platform}-x86_64.zip
 
-wget https://github.com/google/protobuf/releases/download/v${version}/${file}
+wget https://github.com/protocolbuffers/protobuf/releases/download/v${version}/${file}
```

Comment:
the first is redirected to the second, so no need to make this change.

Reply:
hey @yliaog please check this https://github.com/google/protobuf.dart/issues/532#issuecomment-888515218